flow/project.py

```diff
@@ -2468,6 +2452,11 @@ def eligible_for_submission(self, job_operation):
             return False
         return True
 
+    @deprecated(deprecated_in="0.8", removed_in="1.0",
+                details="Use _eligible_for_submission instead.")
```

Comment:
This might not be appropriate to recommend `_eligible_for_submission`. Users shouldn't use private methods, we're just taking this out of the public API.